### PR TITLE
Scaffold single blog post

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: templates/article.liquid [hunk 1]
```diff
@@ -1,18 +1,87 @@
...
+
+<article class="container">
+  <div class="grid-row justify-center">
+    <div class="grid-item item-s-12  item-m-7">
+      <div class="single-post-details margin-bottom-small">
+        <span class="date">{{ article.published_at | date: '%b %e, %Y' }}</span>
+        <h1 class="article-title margin-bottom-small">
+          {{ article.title }}
+        </h1>
+        <p>
```
Comment (varffs) [inline templates/article.liquid:11]: This excerpt should have a 1 (maybe 2) column indent in relation to the tags below

File: templates/article.liquid [hunk 2]
```diff
@@ -1,18 +1,87 @@
...
+    <div class="grid-item item-s-12 item-m-7 single-post-footer">
+      <p class="post-tags">
+        <span class="u-bold-cn">TAGGED: </span>
+        {% for tag in article.tags %}
+          <a href="{{ blog.url }}/tagged/{{ tag }}" rel="tag">{{ tag }}</a>{% if forloop.last != true %},{% endif %}
+        {% endfor %}
+      </p>
+      <p class="post-share">
+        <span class="u-bold-cn">SHARE: </span>
+        <a href="https://twitter.com/home?status={{ article.title | url_encode }}%20via%20%40apt25mx%20{{ shop.url }}{{ article.url }}"><img class="icon" src="{{ 'twitter.svg' | asset_url }}"></a>
```
Comment (varffs) [inline templates/article.liquid:83]: is this better than twitter intent urls?

**Review by varffs — CHANGES_REQUESTED** [review]
General widths dont exactly match the existing site. Maybe we also should take down mid grid size to only 10 col or something and large to even smaller?

File: templates/article.liquid (hunk 1 above)
Comment (mixcas) [reply templates/article.liquid:11]: This will be added whenever this gets styled

File: templates/article.liquid (hunk 2 above)
Comment (mixcas) [reply templates/article.liquid:83]: This is how it is in the current website. If you feel Web Intents are better, open an issue and we can discuss it there cuz I'm not familiarized with them.

**Comment by mixcas** [comment]
This is  comment from waffle **ignore it**

File: templates/article.liquid (hunk 1 above)
Comment (varffs) [reply templates/article.liquid:11]: this is scaffold tho as relies on the grid

File: templates/article.liquid (hunk 2 above)
Comment (varffs) [reply templates/article.liquid:83]: oh I've no idea never seen this before. I've always used https://dev.twitter.com/web/intents but seems pretty similar in result. I wonder if this approach is depreciated but long time still working like fb sharer.php?

**Pull request merged**